modules/api/src/main/scala/com/github/scoquelin/arugula/commands/RedisSortedSetAsyncCommands.scala

```diff
@@ -5,32 +5,365 @@ import scala.concurrent.Future
+   * @param key The key
+   * @param start The start index
+   * @param stop The stop index
+   * @return The members in the specified range
+   */
+  def zRange(key: K, start: Long, stop: Long): Future[List[V]]
+
+  /**
+   * Return a range of members with scores in a sorted set, by index.
+   * @param key The key
+   * @param start The start index
+   * @param stop The stop index
+   * @return The members with scores in the specified range
+   */
   def zRangeWithScores(key: K, start: Long, stop: Long): Future[List[ScoreWithValue[V]]]
+
+  /**
+   * Return a range of members in a sorted set, by score
+   * @param key The key
+   * @param range The range of scores
+   * @param limit Optional limit
+   * @return The members in the specified score range
+   */
   def zRangeByScore[T: Numeric](key: K, range: ZRange[T], limit: Option[RangeLimit]): Future[List[V]]
+
+  /**
+   * Return a range of members in a sorted set, by score, with scores ordered from high to low
+   * @param key The key
+   * @param range The range of scores
+   * @param limit Optional limit
+   * @return The members in the specified score range
+   */
+  def zRangeByScoreWithScores[T: Numeric](key: K, range: ZRange[T], limit: Option[RangeLimit] = None): Future[List[ScoreWithValue[V]]]
+
+  /**
+   * Return a range of members in a sorted set, by score, with scores ordered from high to low
+   * @param key The key
+   * @param range The range of scores
+   * @param limit Optional limit
+   * @return The members in the specified score range
+   */
   def zRevRangeByScore[T: Numeric](key: K, range: ZRange[T], limit: Option[RangeLimit]): Future[List[V]]
+
+  /**
+   * Return a range of members in a sorted set, by score, with scores ordered from high to low
+   * @param key The key
+   * @param range The range of scores
+   * @param limit Optional limit
+   * @return The members in the specified score range
+   */
+  def zRevRangeByScoreWithScores[T: Numeric](key: K, range: ZRange[T], limit: Option[RangeLimit] = None): Future[List[ScoreWithValue[V]]]
+
+  /**
+   * Increment the score of a member in a sorted set
+   * @param key The key
+   * @param amount The amount to increment by
+   * @param value The value
+   * @return The new score of the member
+   */
+  def zIncrBy(key: K, amount: Double, value: V): Future[Double]
+
+  /**
+   * Determine the index of a member in a sorted set
+   * @param key The key
+   * @param value The value
+   * @return The index of the member, or None if the member does not exist
+   */
+  def zRank(key: K, value: V): Future[Option[Long]]
+
+  /**
+   * Determine the index of a member in a sorted set, with the score
+   * @param key The key
+   * @param value The value
+   * @return The index of the member, or None if the member does not exist
+   */
+  def zRankWithScore(key: K, value: V): Future[Option[ScoreWithValue[Long]]]
+
+  /**
+   * Determine the index of a member in a sorted set, with the score
```

Comment:
minor : I think this method would only return the index

Reply:
fixed the docs.